ui/raidboss/data/README.md

```diff
@@ -0,0 +1,151 @@
+# Triggers File Format
+
+## File Structure
+
+```
+[{
+  zoneRegex: /match for the zone/,
+  timelineFile: 'filename.txt',
+  timeline: `hideall`,
+  timelineReplace: [
+    {
+	  locale: 'en',
+	  replaceText: {
+      "regexSearch": "strReplace",
+	  },
+	  replaceSync: {
+      "regexSearch": "strReplace",
+	  },
+	},
+  ],
+  resetWhenOutOfCombat: true,
+  triggers: [
+    { /* ..trigger 1.. */ },
+    { /* ..trigger 2.. */ },
+    { /* ..trigger 3.. */ },
+  ]
+},
+{
+  zoneRegex: /match for another zone/,
+  triggers: [
+    { /* ..trigger 1.. */ },
+    { /* ..trigger 2.. */ },
+    { /* ..trigger 3.. */ },
+  ]
+}]
+```
+# Elements
+
+**zoneRegex**
+Largely self-explanatory. For most situations you will want to match the encounter zone and nothing else.
```

Comment:
Maybe worth saying "zone name as given by ACT", in the sense that it doesn't need to be translated and isn't what's in game.